Keas.Mvc/Controllers/PersonAdminController.cs

```diff
@@ -174,8 +174,10 @@ public IActionResult UploadPeople()
         }
 
         [HttpPost]
-        public async Task<IActionResult> UploadPeople(IFormFile file)
+        public async Task<IActionResult> UploadPeople(IFormFile file, string updateExisting)
```

Comment:
Why not use a bool if you are already using a checkbox?

Reply:
Couldn't get it to bind